src/ui/profile_preferences.ui

```diff
@@ -3479,11 +3482,44 @@ you can use it but there could be issues with aligning columns of text</string>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBox_debugShowAllCodepointProblems">
+            <property name="toolTip">
+             <string>&lt;p&gt;If checked this will cause all problem Unicode codepoints to be reported in the debug output as they occur; if cleared then each different one will only be reported once and summarized in as a table when the console in which they occurred is finally destroyed (when the profile is closed).&lt;/p&gt;</string>
+            </property>
             <property name="text">
              <string>Report all Codepoint problems immediately</string>
             </property>
+           </widget>
+          </item>
+          <item row="6" column="0">
+           <widget class="QLabel" name="label_networkPackerTimeout">
+            <property name="text">
+             <string>Network packet timeout:</string>
+            </property>
+            <property name="buddy">
+             <cstring>spinBox_networkPacketTimeout</cstring>
+            </property>
+           </widget>
+          </item>
+          <item row="6" column="1">
+           <widget class="QSpinBox" name="spinBox_networkPacketTimeout">
             <property name="toolTip">
-             <string>&lt;p&gt;If checked this will cause all problem Unicode codepoints to be reported in the debug output as they occur; if cleared then each different one will only be reported once and summarized in as a table when the console in which they occurred is finally destroyed (when the profile is closed).&lt;/p&gt;</string>
+             <string>&lt;p&gt;On Game Servers which do not use GA &lt;i&gt;go-ahead&lt;/i&gt; signalling this sets how long Mudlet will wait for further network packets before assuming that there is not any more to arrive. The default is 300 milli-seconds. Larger values will help to prevent &lt;i&gt;packet fragmentation&lt;/i&gt; but can make Mudlet seem less responsive to incoming Game text; smaller values may make Mudlet seem to handle such text faster but runs the risk of breaking what is supposed to be a single piece of text or out-of-band game data into multiple chunks. This can break triggers or OOB features.&lt;/p&gt;
```

Comment:
How about the following?

> Go-Ahead (GA) tells Mudlet when the game server is done sending text. On games that don't support GA, this option controls how long Mudlet will wait for more text to arrive. Larger values will help reduce the risk that a large piece of text has unintended linebreaks in the middle of it, which would break triggers. Lesser values will increase the risk of text getting broken up, but will make the game feel more responsive.